Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Self-Enrichment.yml

```diff
@@ -133,32 +128,31 @@ tasks:
     isautoswitchedtoquietmode: false
   "88":
     id: "88"
-    taskid: 5e18d026-6149-4f5c-858f-c7995a5b3550
+    taskid: 41e5b734-11c8-45d6-83cd-dd04e1f3980b
     type: regular
     task:
-      id: 5e18d026-6149-4f5c-858f-c7995a5b3550
+      id: 41e5b734-11c8-45d6-83cd-dd04e1f3980b
       version: -1
       name: Search external service information
-      description: Get a list of all your external services filtered by the remote IP of the alert.
-      script: '|||asm-list-external-service'
+      description: Get service details according to the service ID.
```

Comment:
Is `asm-get-external-service` going to hang, does it need a sleep or poll?

Reply:
Was hoping that https://jira-dc.paloaltonetworks.com/browse/EXPANDR-8203 should make it so this isn't necessary.  We can add it later if needed